packages/storybook/stories/additional-docs.js

```diff
@@ -155,7 +155,9 @@ export const additionalDocs = {
     maturityCategory: USE,
     maturityLevel: DEPLOYED,
   },
-  'va-date-text-input': {
+  'va-memorable-date': {
+    guidanceHref: 'form/memorable-date',
+    guidanceName: 'Memorable Date',
```

Comment:
We don't need to specify a guidance name for web components unless we want it set to something other than the component name.

https://github.com/department-of-veterans-affairs/component-library/blob/master/packages/storybook/stories/wc-helpers.jsx#L230-L235